qa/setup_dali_extra.sh

```diff
@@ -2,7 +2,7 @@
 
 # Fetch test data
 export DALI_EXTRA_PATH=${DALI_EXTRA_PATH:-/opt/dali_extra}
-export DALI_EXTRA_URL=${DALI_EXTRA_URL:-"https://github.com/NVIDIA/DALI_extra.git"}
+export DALI_EXTRA_URL="https://github.com/a-sansanwal/DALI_extra.git"
```

Comment:
The changes in DALI_extra have been merged already. You can now revert the change in this file